src/probeinterface/generator.py

```diff
@@ -99,26 +104,26 @@ def generate_multi_columns_probe(
     xpitch: float = 20,
     ypitch: float = 20,
     y_shift_per_column: Optional[np.array | list] = None,
-    contact_shapes: str = "circle",
+    contact_shapes: "circle" | "rect" | "square" = "circle",
```

Comment:
The use of  `|` still requires `from __future import annotations` for Python3.8

Reply:
This one already has `from __future__ import annotations` :)